apple/Sources/FerrostarSwiftUI/Views/SpeedLimit/ROWSpeedLimit.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+public struct ROWSpeedLimitView: View {
```

Comment:
Aside: some countries change various parts of this; we can expose those as customizations later on if desired, but don't need to for this PR. Just trying to be a bit less US-centric than Mapbox while also getting the terminology more correct 😂 

```suggestion
/// The common speed circular limit sign with a white background and red border.
/// Also known as the Vienna Convention Sign C14.
public struct ViennaConventionStyleSpeedLimitView: View {
```